tests/unit/expire.tcl

```diff
@@ -834,7 +834,7 @@ start_server {tags {"expire"}} {
     } {} {needs:debug}
 }
 
-start_cluster 1 0 {tags {"expire external:skip cluster slow"}} {
```

Comment:
let's mention this in the top comment

Reply:
yes, i actually edited and updated the top comment after i pushing the code.